website/static/templates.json

```diff
@@ -2194,5 +2194,68 @@
       "langchain",
       "new"
     ]
+  },
+  {
+    "title": "Azure Functions C# HTTP Trigger using Azure Developer CLI",
+    "description": "This repository contains an Azure Functions HTTP trigger quickstart written in C# and deployed to Azure Functions Flex Consumption using the Azure Developer CLI (azd). The sample uses managed identity and a virtual network to make sure deployment is secure by default.",
+    "preview": "./templates/images/test.png",
+    "website": "https://github.com/Azure-Samples",
+    "author": "Azure Functions Team",
+    "source": "https://github.com/Azure-Samples/functions-quickstart-dotnet-azd",
+    "tags": ["new", "functions", "flex", "dotnetCsharp", "http", "entra-id", "vnet", "appinsights", "msft", "bicep"]
```

Comment:
Make sure the tags you are using exists here: https://github.com/Azure/awesome-azd/blob/main/website/src/data/tags.tsx#L135

For example, change `vnet` to `vnets` :https://github.com/Azure/awesome-azd/blob/main/website/src/data/tags.tsx#L512

Remove `flex` or add a new tag `flex` to the tags file.

Reply:
Fixed